packages/nextjs/app/admin/_components/GrantReview.tsx

```diff
@@ -61,6 +62,17 @@ export const GrantReview = ({ grant, selected, toggleSelection }: GrantReviewPro
 
   const { handleReviewGrant, isLoading } = useReviewGrant(grant);
 
+  // Fetch all grants for this builder to show count and detail in tooltip
+  const { data: grants, error } = useSWR<GrantData[]>(
+    grant.status === PROPOSAL_STATUS.PROPOSED || grant.status == PROPOSAL_STATUS.SUBMITTED
```

Comment:
Nitpick: We could remove this check, since whenver we are rending <GrantReivew /> this condition will always be true 🙌

Reply:
Fixed!